www/js/index.js

```diff
@@ -136,6 +136,12 @@ function findLine(e) {
     lineFound.click();
   }
 }
+function feelingLucky() {
+  if (search.$results.childNodes.length === 1) {
+    const singleResult = document.getElementsByClassName('panktee search-result');
```

Comment:
I would do `document.querySelector('#results .search-result').click();`  incase we use that class somewhere else in the future, better to get the element with the idea and search within that.